### PR TITLE
Update personal profile page and Developer Guide for v1.1

### PR DESCRIPTION
**Pull request by Preshita01**

### CODE REVIEW

File: docs/team/preshita01.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    * As the team member in-charge of documentation, I am responsible for ensuring that the various project documents are up to standard (e.g. Developer guide, User Guide). 
+
+* **Documentation**:
+    * Developer Guide:
+      * Added on to the target user profile, value proposition, user stories, use cases, non-functional requirements and glossary.
+    * User Guide:
+        * Added the documentation for the features `add` and `delete`
+    * README:
+        * Added the overview and acknowledgement sections
+        * Added the image of TutorAid's GUI
```
Comment (JunWei3112) [inline docs/team/preshita01.md:22]: LGTM

**Review by JunWei3112 — APPROVED** [review]

File: docs/team/preshita01.md (hunk 1 above)
Comment (dt-td) [reply docs/team/preshita01.md:22]: LGTM

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -254,74 +254,353 @@ _{Explain here how the data archiving feature will be implemented}_
...
+   
+    *  1a1. TutorAid displays an error message highlighting the wrongly formatted command and requests the tutor to re-enter the command.
+       
+    *  1a2. User re-enters the command.
+
+        Steps 1a1-1a2 are repeated until the data entered are correct.
+
+        Use case resumes from step 2.
+    
+*  2a. TutorAid detects a wrongly formatted command to edit the pricing of a class (e.g. missing fields in the component, typos in the command)
```
Comment (wpinrui) [inline docs/DeveloperGuide.md:552]: Do you think if it is okay for us to put all the invalid input together as one extension? At least if the input comes from the same step in the MSS

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -254,74 +254,353 @@ _{Explain here how the data archiving feature will be implemented}_
...
+   
+    *  2c1. TutorAid displays an error message highlighting the invalid price and requests the tutor to re-enter the command.
+       
+    *  2c2. User re-enters the command with another date.
+
+        Steps 2c1-2c2 are repeated until the command entered is correct.
+
+        Use case resumes from step 3.
+
+### Non-Functional Requirements
```
Comment (wpinrui) [inline docs/DeveloperGuide.md:582]: LGTM

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -254,74 +254,353 @@ _{Explain here how the data archiving feature will be implemented}_
...
+7.  The system should only be able to handle up to 1000 students.
+8.  The user input elements are still visible even with hundreds of data records being displayed.
+9.  The project is expected to adhere to a schedule that delivers updates every 2 weeks.
+10.  The product should be fully functional offline and should not rely on internet connection to carry out any of its functions.
+11.  The product is not required to handle mobile phone capabilities.
+12.  The product is not required to handle any other languages (other than English).
+13.  The product is not required to facilitate monetary transactions for payment of tuition fees.
+14.  The product data should be transferable from one computer to another.
+
+### Glossary
```
Comment (wpinrui) [inline docs/DeveloperGuide.md:598]: LGTM

File: docs/team/preshita01.md [hunk 5]
```diff
@@ -9,14 +9,11 @@ TutorAid is a desktop application used for helping tech-savvy private tutors, wh
 
 Given below are my contributions to the project.
 
-* **Documentation in-charge**
-    * As the team member in-charge of documentation, I am responsible for ensuring that the various project documents are up to standard (e.g. Developer guide, User Guide). 
-
 * **Documentation**:
+    * User Guide:
```
Comment (wpinrui) [inline docs/team/preshita01.md:13]: LGTM

**Review by wpinrui — APPROVED** [review]

**Pull request merged**